docs/tenancy/1.x/hooks-custom.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Custom lifecycle hooks
+icon: fal fa-gear
+excerpt: >
+    How to create custom lifecycle hooks
+tags:
+    - hooks
+    - tenant
+---
+
+# Lifecycle Hooks: Custom Hooks
+
+- [Overview](#overview)
+- [Base](#base)
+- [Implementation](#implementation)
+  - [General](#general)
+  - [Differences](#differences)
+- [Registration](#registration)
+- [Advance](#advance)
+  - [ConfiguresHook Event](#configureshook-event)
+
+## Overview
+
+**Purpose**
+
+Enable custom functionality that isn't available though existing hooks
+
+**Requirements**
+
+Knowledge of how [Lifecycle Hooks work](hooks-general)
+
+## Base
+
+You can build your own hook by implementing the `Tenancy\Contracts\LifecycleHook` contract.
+
+There are two abstract Hook classes that make implementation easier:
+
+- `Tenancy\Lifecycle\Hook`
+- `Tenancy\Lifecycle\ConfigurableHook`
+
+The basic difference is that the hook settings are not expected change in a `Hook`,
+but are expected to be fluid in the `ConfigurableHook`.
+
+I.E. A Lifecycle Hook that always run at the same priority on the same queue should use `Hook`,
+But a LifecycleHook that will run on the Tenant's Queue should use `ConfigurableHook`
```

Comment:
You ended the last sentence with a command, then you should not put a capital later on.